scriptlets/test-executions-rerunner/test_executions_rerunner.py

```diff
@@ -8,14 +8,28 @@
 
 import logging
 import re
+from functools import partial
 from os import environ
 from typing import Optional
 
-import requests
+from requests import Session
+from requests.adapters import HTTPAdapter
 from requests.auth import HTTPBasicAuth
+from urllib3.util import Retry
 
 logging.basicConfig(level=logging.INFO)
 
+
+requests = Session()
+retries = Retry(
+    total=3,
+    backoff_factor=0.1,
```

Comment:
I'm wondering if we should set `backoff_factor` to a bigger number (for example `backoff_factor=3`, so we will have jitter as 5, 10, 20) to give it more chance to get through the gateway-related errors that might not be recovered shortly?

Reply:
Good idea